src/page/order_pcbs.jsx

```diff
@@ -36,6 +39,22 @@ let OrderPcbs = React.createClass({
               <semantic.Menu.Item as="a" href={aislerUrl}>
                 <img src="/images/aisler.png" />
               </semantic.Menu.Item>
+
+              <semantic.Menu.Item>
+                <h4>PCB Comparison</h4>
+              </semantic.Menu.Item>
+              <semantic.Menu.Item
+                as="a"
+                href={pcbshopperUrl}
+                onClick={() => {
+                  if (!this.state.downloaded) {
+                    window.open(zipPath)
+                  }
+                }}
+              >
+                <img src="/images/pcbshopper.png" />
+              </semantic.Menu.Item>
```

Comment:
This is still in the wrong place and the addition of the text "PCB Comparison" makes for quite a confusing interface. 

![image](https://user-images.githubusercontent.com/206854/55076239-d888d380-508c-11e9-85df-0bc7931f8a39.png)

I will go over this with Jon today and we can sketch out how we want it to look but PCBWay should definitely stay alongside Aisler and not be at the other end. 

Another issue is that the responsive trigger point is no longer correct as the menu is wider.

![image](https://user-images.githubusercontent.com/206854/55076520-8c8a5e80-508d-11e9-8208-6718d1c0d488.png)